# Grasp service: Open loop parametrized grasp strategy with early abort

float64 x
float64 y
float64 z_lim
float64 up
float64 down
int64 nFlips
float64 oscillationAmplitude
int64 id
float64 handSpeed
---
int64 error
string logFileName
int64 ret
string msg

